curp-testutils/src/test_cmd.rs

```diff
@@ -43,15 +43,29 @@ impl Display for ExecuteError {
 
 #[derive(Serialize, Deserialize, Clone, Debug, PartialEq)]
 pub struct TestCommand {
-    id: ProposeId,
-    keys: Vec<u32>,
+    pub id: ProposeId,
+    pub keys: Vec<u32>,
```

Comment:
Can we provide functions to access these field?

Reply:
Sorry the `pub` is actually not needed as the public `Commad` trait provides these get funtions. 

Fixed.